mlir/tools/rocmlir-gen/rocmlir-gen.cpp

```diff
@@ -2132,10 +2138,13 @@ static func::FuncOp createVerifierFunc(ModuleOp module, const KernelIF &kernel,
   std::string verifyFuncName = "mcpuVerify";
   if (valElemType.isF32()) {
     verifyFuncName += "Float";
+  } else if (valElemType.isInteger(8)) {
```

Comment:
Why not have that last case (with the `to_string`) cover these two special cases?